Makefile

```diff
@@ -9,7 +9,12 @@ TEST_CONFIG ?= tests/ansible.cfg
 test-minimal: TEST_OUTFILE := tests/logs/test_minimal_out_$(shell date +%FT%T%Z).log
 test-minimal:
 	mkdir -p tests/logs
-	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
+	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e ospdo_src=True -e @$(TEST_VARS) -e @$(TEST_SECRETS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
```

Comment:
Should it be ospdo_src=False as it is under minimal?

Reply:
that should be removed , no need to add the variable as false as the tag is checking if it's defined , the bool check is not a must, and I didnt't want to add unnecessary code to the regular test-minimal.